apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -11,8 +11,7 @@ import ch.tutteli.kbox.glue
  * Expects that the subject of the assertion is (equal to) [expected].
  *
  * @return An [Expect] for the current subject of the assertion.
- * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
- *
+
```

Comment:
I think you forgot to add the `*`, shouldn't it be 
```suggestion
 *
```

Reply:
You are right. :disappointed:

Reply:
fixed it.